docs/rst/user_manual/getting_started.rst

```diff
@@ -0,0 +1,102 @@
+.. include:: ../exports/alias.include
+
+.. _getting_started_tutorial:
+
+###############
+Getting Started
+###############
+
+|ddsrouter| is a cross-platform non-graphic application developed by eProsima and powered by Fast DDS
+that allow to create a communication bridge that connects two DDS networks that otherwise would be isolated.
+The main use case of the |ddsrouter| is to communicate two DDS networks that are physically or virtually separated
+and belong to different LANs, allowing those entities to publish and subscribe to local and remote topics indistinctly.
+
+
+Project Overview
+================
+
+The |ddsrouter| is an application that internally run :term:`Participants <Participant>`, that are an abstraction of
+DDS :term:`DomainParticipants <DomainParticipant>`.
+Each one of these Participants is an communication interface: a "door" to a specific DDS network configuration.
+These Participants allow the application to connect to different DDS networks at the same time.
+Every time one of these Participants receives a message from the DDS network it is connected, it will forward the
+data and the source of this message through the other Participants.
+In which DDS configurations and which topics the |ddsrouter| will work depends on the initial
+:ref:`DDS Router configuration <user_manual_configuration>`.
+
+The following schema represents a |ddsrouter| local use case.
+In this scenario, we have different DDS networks that are isolated one to each other, due to Transport Protocol
+(UDP, TCP, etc.), Discovery Protocol (Simple, Discovery Server, etc.) or by DDS :term:`Domain Id`.
+Configuring the |ddsrouter| to have 4 different Participants, each of them using one of the different scenario
+configurations, will create internally 4 Participants.
+All the data that arrives to one of the Participants will be forwarded through the others, allowing all the machines
+to connect to each other independently of their different configurations.
+
+.. figure:: /rst/figures/ddsrouter_overview.png
+
+
+WAN Communication
+-----------------
+
+In order to achieve a WAN communication of two networks that work in different LANs, each LAN requires a running
+|ddsrouter| application.
+These |ddsrouter| will communicate to each other using DDS over WAN, and will route every message received in LAN
+to the remote |ddsrouter|.
+Once the remote Router has received it, it will transmit the data to the local networks it is connected with.
+This way, both DDS networks will behave as if they would belong to the same LAN.
```

Comment:
```suggestion
Once the remote Router receives data, it will transmit it to the local networks to which it is connected.
This way, both DDS networks will behave as if they would belong to the same LAN.
```

Reply:
:+1: